library/cartridge_connect_to_membership.py

```diff
@@ -4,7 +4,7 @@
 
 argument_spec = {
     'console_sock': {'required': True, 'type': 'str'},
-    'hostvars': {'required': True, 'type': 'dict'},
+    'facts': {'required': True, 'type': 'dict'},
```

Comment:
The `hostvarts` name is better than `facts` because it reflects that is't "vars by hosts".

Reply:
`module_hostvarts` is better